static/js/notifications.js

```diff
@@ -156,24 +157,37 @@ exports.redraw_title = function () {
+
+        rendered_favicon = render_favicon({
+            count_path,
+            border_path,
+        });
+    } else {
+        rendered_favicon = render_favicon();
     }
+    const svg = $(rendered_favicon).find("#favicon").prevObject[0];
```

Comment:
There’s no need to parse the SVG string with `$()` only to extract the SVG string back out again with `.outerHTML`.